R/read_redcap.R

```diff
@@ -75,6 +78,7 @@ read_redcap <- function(redcap_uri,
                         raw_or_label = "label",
                         forms = NULL,
                         export_survey_fields = TRUE,
+                        export_data_access_groups = TRUE,
```

Comment:
Wondering, should `export_survey_fields` and `export_data_access_groups` be `TRUE` or `NULL`, where `NULL` means that export them if they are there?

Reply:
This follows the REDCapR conventions again, I'm impartial but there is some sense in keeping in line with REDCapR since these get directly copied from it (although in REDCapR they're default `FALSE`).

Reply:
The only reason would be to fail more loudly if someone expected DAGs (or survey fields), intentionally set the field to `TRUE` and then receives an informative error message if the expected DAG or survey fields aren't there. Admittedly this is probably a rare edge case but on the other hand I think it would be good to implement a consistent implementation of these sorts of flags across all functions of the package. Might be worth discussing at standup.

Reply:
TODO: implement default is `NULL` which returns if detected, fails if specified `TRUE` and not detected.